### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'codePoint')

### ISSUE THREAD

**Issue by Kristalkill** [EvgenyOrekhov/eslint-config-hardcore#444]
**Line**        `text.replace(/[^\p{Letter}\p{Number}]\p{Emoji_Modifier_Base}\p{Emoji_Modifier}\p{Emoji_Presentation}\p{Emoji}\uFE0F/gu, ' ')`
**Linter crash error**      
```
TypeError: Cannot read properties of undefined (reading 'codePoint')
Occurred while linting D:\My Files\END\Projects\Discord\Bot\END\src\bot\Structures\Core\Utils\String.ts:4
Rule: "optimize-regex/optimize-regex"

TypeError: Cannot read properties of undefined (reading 'codePoint')
Occurred while linting D:\My Files\END\Projects\Discord\Bot\END\src\bot\Structures\Core\Utils\String.ts:4
Rule: "optimize-regex/optimize-regex"
    at getSortValue (D:\My Files\END\Projects\Discord\Bot\END\node_modules\regexp-tree\dist\optimizer\transforms\char-class-classranges-merge-transform.js:103:26)
    at sortCharClass (D:\My Files\END\Projects\Discord\Bot\END\node_modules\regexp-tree\dist\optimizer\transforms\char-class-classranges-merge-transform.js:63:16)
    at Array.sort (<anonymous>)
    at Object.CharacterClass (D:\My Files\END\Projects\Discord\Bot\END\node_modules\regexp-tree\dist\optimizer\transforms\char-class-classranges-merge-transform.js:35:17)
    at pre (D:\My Files\END\Projects\Discord\Bot\END\node_modules\regexp-tree\dist\traverse\index.js:218:55)
    at visit (D:\My Files\END\Projects\Discord\Bot\END\node_modules\regexp-tree\dist\traverse\index.js:41:13)
    at visit (D:\My Files\END\Projects\Discord\Bot\END\node_modules\regexp-tree\dist\traverse\index.js:80:15)
    at visit (D:\My Files\END\Projects\Discord\Bot\END\node_modules\regexp-tree\dist\traverse\index.js:88:15)
    at astTraverse (D:\My Files\END\Projects\Discord\Bot\END\node_modules\regexp-tree\dist\traverse\index.js:99:3)
    at Object.traverse (D:\My Files\END\Projects\Discord\Bot\END\node_modules\regexp-tree\dist\traverse\index.js:172:5)
Process finished with exit code -1
```

**Comment by RedGuy12** [comment]
This is a bug in https://github.com/BrainMaestro/eslint-plugin-optimize-regex

**Comment by EvgenyOrekhov** [comment]
I removed [eslint-plugin-optimize-regex](https://github.com/BrainMaestro/eslint-plugin-optimize-regex) since we have [`unicorn/better-regex`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/better-regex.md) which is better and less buggy. This will be included in the next release.

**Comment by EvgenyOrekhov** [comment]
Released in [v24.0.0](https://github.com/EvgenyOrekhov/eslint-config-hardcore/releases/tag/v24.0.0).
